src/finput.js

```diff
@@ -197,6 +202,11 @@ class Finput {
     this.setValue(newValue, false);
   }
 
+  isModifierKeyPressed(originalEvent) {
+    const isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
+    return isMac ? originalEvent.metaKey : originalEvent.ctrlKey;
```

Comment:
Do we need to check for the existence of the Mac platform or can we always just use the metaKey property over the ctrlKey one?